tests/test_prompt.py

```diff
@@ -288,6 +288,9 @@ def test_json3():
     prompt = Prompt()
     with pytest.raises(MemorValidationError, match=r"Invalid prompt structure. It should be a JSON object with proper fields."):
```

Comment:
This test case is not incorrect, but it is not strong for this purpose; it's better to add a test case that tries to load a partially correct JSON object.